src/app/settings/settings.component.html

```diff
@@ -102,11 +102,18 @@ <h5>SETTINGS</h5>
             <div class="form-group row">
               <label for="RemoveOldSlowLogs" class="col-sm-5 col-form-label">Remove old slow logs</label>
               <div class="col-sm-3">
-                <p class="form-control-static" id="RemoveOldSlowLogs">{{ agentConf?.qan?.RemoveOldSlowLogs ? 'ON' :
+                <p class="form-control-static" id="RemoveOldSlowLogs">{{ agentConf?.qan?.SlowLogsRotation ? 'ON' :
```

Comment:
RemoveOldSlowLogs vs SlowLogsRotation 
Can we use the same name? because this will be confusing